src/main/java/de/hs_mannheim/informatik/ct/util/CookieManager.java

```diff
@@ -0,0 +1,150 @@
+package de.hs_mannheim.informatik.ct.util;
+
+/*
+ * Corona Tracking Tool der Hochschule Mannheim
+ * Copyright (C) 2021 Hochschule Mannheim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import lombok.val;
+import org.springframework.web.util.WebUtils;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.time.LocalTime;
+
+public class CookieManager {
+    private HttpServletRequest request;
+    private HttpServletResponse response;
+
+    public CookieManager(HttpServletRequest request){
+        this.request = request;
+    }
+
+    public CookieManager(HttpServletRequest request, HttpServletResponse response){
+        this(request);
+        this.response = response;
+    }
+
+    public static enum Cookies{
+        CHECKED_IN_EMAIL("checkedInEmail");
+
+        private String name;
+
+        private Cookies(String name) {
+            this.name = name;
+        }
+
+        public String getName(){
+            return this.name;
+        }
+    }
```

Comment:
For each cookie type an enum should be defined with its cookie name (Other cookie types should be integrated soon)